getting_started.Rmd

```diff
@@ -74,80 +83,89 @@ In the pop up window, search for and add `jhudsl-robot`.
 
 Choose the `write` option then click `Add jhudsl-robot to this repository`.
 
+TODO: I don't see the write option.
+
+
 ### 6. In personal repositories: check your settings
 
-If you have a personal repository, or a organization that hasn't used Github Actions before, you'll also need to check that workflow actions have been enabled.
+TODO: Clarify
```

Comment:
Can you expand on what needs to be clarified here?

Reply:
I think the title of this step, "In personal repositories: check your settings" is a bit confusing. Maybe we can name it to "Check if GitHub Actions have been enabled"?

Reply:
Whatever wording you think would be more clear or if a screenshot would help…